csAudioApps/stepSeq
# Sockets

## Description:
- This update sets up the initial socket connection between the server and front-end to send and receive a string
## Changes I Made:
- App.jsx sets up socket connection, sends a message and listens for messages
- Server.js opens socket with listener to send message
## How to Test:
- npm install and start
- write a message in the input and send, message should display in the console of any open clients and the server console
